OC4IDS_Data_Feedback_Notebook.ipynb

```diff
@@ -892,6 +892,18 @@
       "execution_count": null,
       "outputs": []
     },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "UhHQvefAEQoO"
+      },
+      "source": [
+        "df = pd.DataFrame(project_status)\n",
+        "df"
```

Comment:
The `%%sql` magic's `autopandas` config is set to `True` in the [setup cell](https://colab.research.google.com/github/open-contracting/notebooks-oc4ids/blob/data_feedback_tables/OC4IDS_Data_Feedback_Notebook.ipynb#scrollTo=VjuPJaBUMapU&line=7&uniqifier=1) so it already returns Pandas DataFrames rather than regular resultsets.

As such, you can just use the name of the dataframe to display the table:

```suggestion
        "project_status"
```

The same applies to the other changes in this PR.